specs/protocol/system-config.md

```diff
@@ -120,6 +122,18 @@ manner, it is stored at a special storage slot corresponding to
 Unlike the other values, the `unsafeBlockSigner` only operates on blockchain
 policy. It is not a consensus level parameter.
 
+## System config contents (version x+interop)
```

Comment:
Diffs for interop should be put into the `interop` dir, with matching file-name `system-config.md`